### ISSUE TITLE
Display domains of blocked requests in doorhanger?

### ISSUE THREAD

**Issue by pdehaan** [mozilla/blok#119]
Pretty sure this has come up before, but now that we're a doorhanger and not a pseudo toolbar UI, it may be nice to tell me *what* has been blocked, not just that there were "blocked tracker requests".

I'm not sure what is or isn't working on this page, but telling me the top level domains for blocked requests would be super helpful in determining if something "feels wrong" (versus just blocking google analytics pings or whatever).

![firefox_test_pilot_-_blok](https://cloud.githubusercontent.com/assets/557895/17902779/8a75a7d0-691d-11e6-9ad8-efd7e3b3c221.png)

**Comment by groovecoder** [comment]
FWIW I'd like to defer UI additions & enhancements like this until after I've implemented Redux+React or a similar JS architecture. The current JS for the UI is flaky (see https://github.com/mozilla/blok/issues?q=is%3Aissue+is%3Aopen+label%3Abug) and I'm just starting to refactor into a stable architecture.

**Comment by johngruen** [comment]
@groovecoder wfm, i'll see if aislinn can do some mocks for this though...would be a cool after-launch feature

**Comment by ChenMorpheus** [comment]
Please find the link below for our proposal, and let me know if any questions, thanks.
https://mozilla.invisionapp.com/share/GK8ZKDR75#/196906617_Proposal_1

**Comment by groovecoder** [comment]
@pdolanjski and @johngruen want to take a look at this UI proposal?

**Comment by johngruen** [comment]
@groovecoder this is a stretch feature right now, how long do you think it would take to implement?

**Comment by groovecoder** [comment]
I tend to estimate things in 1-hour, 1-day, 1-week, 1-month increments. This would be a 1-week change to make and then do all the QA + release work too.

**Comment by pdolanjski** [comment]
I agree that this would be useful for the TestPilot experiment to help diagnose when there are issues.

**Comment by groovecoder** [comment]
The current tools available to diagnose issues ...

The browser JS console *does* show the domains of blocked requests when running the add-on in "dev mode".

The [Blok: Executive Summary dashboard](https://sql.telemetry.mozilla.org/dashboard/blok-executive-summary) (now with up-to-date data!) shows breakage reported by both origin domain and the tracker domains.

All that to say: this change would make diagnosing issues easier, but whom do we expect to be diagnosing issues? The broad user-base, or devs? Because devs can already access the domains of blocked requests in the console, and Mozilla devs can access the Telemetry reports too.

**Comment by pdolanjski** [comment]
Fair enough.  I see it as useful for the reporter to sort of confirm that they may be seeing breakage, particularly if you notice common urls that have broken them before.  So, the usefulness may depend on how common those urls are and will the reporter associate them with previous breakage.  That said, I don't think it's critical.

**Comment by johngruen** [comment]
I'm going to keep this open and just give it a Stretch label

**Comment by ChenMorpheus** [comment]
Could someone give us the update for this ticket? Thanks.
